src/_P123_SI7013.ino

```diff
@@ -0,0 +1,165 @@
+#include "_Plugin_Helper.h"
+
+#ifdef USES_P123
+#include "src/PluginStructs/P123_data_struct.h"
+//#######################################################################################################
+//######################## Plugin 123 SI7013 I2C Temperature Humidity Sensor  ###########################
+//#######################################################################################################
+// 08-06-2019 Florin Muntean
+
+#define PLUGIN_123
+#define PLUGIN_ID_123        123
```

Comment:
Plugin ID 123 is already reserved, see https://github.com/letscontrolit/ESPEasy/issues/3839
I think this one should be **P122**, as AFAICS, this sensor is compatible with the SHT2x series of sensors, according to https://github.com/letscontrolit/ESPEasy/issues/2417 (and I was actually investigating (last night) to pick that one, as the 'planned' developer/requester said he wouldn't be able to complete this, but you are very welcome to take this 👍)

Reply:
let me first get this one going then I can see about the merge. 
I don't think I have a SH2x sensor that I can test with.
This plugin was tested for over 1 year now in my setup.
I propose I do this in two steps for now:
1. Get the SI7013 plugin using P122
2. Make it work with SHT2x (I might have to order some sensors too)

Let me know if you want me to rename the sensor to P122

Reply:
FYI the SI7013 ha much more capabilities including a differential analog input comparing to SHT2x and they share the same I2C address

Reply:
> Let me know if you want me to rename the sensor to P122

P122 is reserved for SHT2x, and is supposed/suggested to also support SI7013, using the library mentioned in issue 2417 : [Rob's SHT2x library](https://github.com/RobTillaart/SHT2x)
If your intention is to also support that SHT2x sensor, then using P122 should be the way to go.
If only replacing/upgrading the implementation for SI7013/SI7021 then you'd better replace/improve P014, as that already supports this sensor, there's no need to have a duplicate.

Reply:
I am not ready for mixing two sensors right now (while basically all have temp and humidity some have extra capabilities and I don't know how to support these safely in the current plugin) can you give me a plugin ID that I can use for SI7013 ?
Combining them seems too risky for me right now given I am also asked to do many other changes. 
I tested current code for over 1 year now.

Reply:
Well if you don't want to mix SHT2x and this one in the same plugin (which I totally understand), why not then upgrade the existing plugin for the SI7013?
I don't think we should have 2 plugins for the same sensor as that's very confusing.  (seen it happen on the OLED and Framed OLED plugin)

Reply:
the only plugin that is closer is SI7021 but SI7013 adds ADC capabilities and I am using those in the plugin

Reply:
See also the BME280 plugin.
It also supports the BMP280 which only measures Temp/Pressure, where the BME280 does also measure humidity.

The SenseAir plugin also supports several sensors which may not all have the same number of sensors on board, but you can still change to not only measure CO2, but also temperature.

The AccuEnergy plugin has even more flexibility in selecting what unit of measure to set on the output, as does the GPS plugin.

Reply:
ok. I will kill this pull request and I will work on making the P014 to work for both sensors